<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>templateChooser</class>
 <widget class="QDialog" name="templateChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Template Chooser</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="default_modle">
      <attribute name="title">
       <string>defalut_modle</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QTextEdit" name="modletext0">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modle1">
      <attribute name="title">
       <string>modle1</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="modletext1"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modle2">
      <attribute name="title">
       <string>modle2</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="modletext2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modle3">
      <attribute name="title">
       <string>modle3</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTextEdit" name="modletext3"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modle4">
      <attribute name="title">
       <string>modle4</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTextEdit" name="modletext4"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modle5">
      <attribute name="title">
       <string>modle5</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QTextEdit" name="modletext5"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>templateChooser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>templateChooser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
